llap-server/src/java/org/apache/hadoop/hive/llap/io/encoded/OrcEncodedDataReader.java

```diff
@@ -523,14 +528,17 @@ private void cleanupReaders() {
         // Ignore.
       }
     }
-    if (rawDataReader != null && isRawDataReaderOpen) {
+    if (rawDataReader != null) {
```

Comment:
no need for null check
````
IOUtils.closeQuietly(rawDataReader);
rawDataReader = null;
````

Reply:
Nice thanks! Done, changed all other classes/interfaces to be Closeable too.